.github/workflows/linux_arm64.yml

```diff
@@ -122,8 +122,8 @@ jobs:
     if: ${{ 'false' }}
     uses: ./.github/workflows/job_debian_packages.yml
     with:
-      runner: 'aks-linux-4-cores-8gb-arm'
-      image: 'openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04'
+      runner: 'ubuntu-22.04'
+      image: 'ubuntu:20.04'
```

Comment:
The gha image could be updated silently we need to lock the particular version

Reply:
It is not a GHA image, it is a Docker Hub image, and it should not be updated silently except for security updates.